pkg/controller/direct/securesourcemanager/mapper.go

```diff
@@ -58,3 +61,37 @@ func OperationMetadata_EndTime_ToProto(mapCtx *direct.MapContext, in *string) *t
 	mapCtx.Errorf("OperationMetadata_EndTime_ToProto not implemented")
 	return nil
 }
+func SecureSourceManagerRepositorySpec_FromProto(mapCtx *direct.MapContext, in *pb.Repository) *krm.SecureSourceManagerRepositorySpec {
+	if in == nil {
+		return nil
+	}
+	out := &krm.SecureSourceManagerRepositorySpec{}
+	// MISSING: Name
+	// MISSING: Description
+	if in.GetInstance() != "" {
+		out.InstanceRef = &krm.SecureSourceManagerInstanceRef{External: in.GetInstance()}
```

Comment:
@gemmahou  I moved this function here, to rename the instance ref from `&refs.*SecureSourceManagerInstanceRef` to `&krm.SecureSourceManagerInstanceRef`

Reply:
Done. Although having a manual change in mapper.generated means any future generate-mapper commands override the change, I am guessing that is the motivation for https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/docs/develop-resources/deep-dives/3-add-mapper.md#to-change-a-method

Reply:
That's expected for now, we will review if there are future changes to the mappers.